feat(QBreadcrumbs): improvement for breadcrumb elements

pdanpdan opened the pull request:
- add marker classes on breadcrumb el parent for when it's disabled/inactive or last
- don't apply activeClass on disabled breadcrumb els
- really disable last breadcrumb el (instead of using pointer-events)
- add visual indicator for active el when kbd navigating

rstoenescu reviewed (changes requested):
Please adapt from https://github.com/quasarframework/quasar/commit/925ba772a8f9c4b486dd614b283f0e5d6563559b

pdanpdan committed:
feat(QBreadcrumbs): improvement for breadcrumb elements

- add marker classes on breadcrumb el  for when it's disabled/inactive
- don't apply activeClass on disabled breadcrumb els
- leave the last breadcrumb enabled
- add visual indicator for active el when kbd navigating

rstoenescu committed:
Update QBreadcrumbsEl.js
